.github/workflows/updatedocs.yml

```diff
@@ -0,0 +1,35 @@
+name: Update Docs
+
+on:
+  schedule:
+    - cron: "0 4 * * *"
+
+jobs:
+  update-docs:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v1
+      - uses: actions/setup-python@v1
+        with:
+          python-version: "3.x"
+      - name: Install requests
+        run: pip install requests
+      - name: Install semantic_version
+        run: pip install semantic_version
```

Comment:
We can create a `requirements.txt` file with the contents:
```
requests
semantic_version
```

and then just run `pip install -r requirements.txt` here. It's a bit cleaner.